core/templates/dev/head/pages/exploration_editor/translation_tab/add_audio_translation_modal_directive.html

```diff
@@ -9,7 +9,8 @@ <h3>Add Audio Translation</h3>
         <div class="form-group">
           <label for="audioTranslationFileField" class="col-lg-2 col-md-2 col-sm-2">Audio File</label>
           <div class="col-lg-10 col-md-10 col-sm-10">
-            <audio-file-uploader id="audioTranslationFileField"
+            <audio-file-uploader class = "uploadAudio"
```

Comment:
Aren't classes typically hyphenated? (separate words using -).

Reply:
@nithusha21 This change was not required. We don't need to introduce a new class name here. Reverting this change.